### ISSUE TITLE
Partie illimitée (le seigneur du mal repoussé ad vitam eternam!)

### ISSUE THREAD

**Issue by Attila2129** [JulienNigon/SpellcasterUniversity-Bug-Report#1180]
Bonjour à tous, je suis en train de jouer sur la carte du lac, et mes étudiants ont résolu le problème du seigneur du mal... à coup de signe des très anciens. A chaque fois que l'un d'eux finit ses études, le seigneur du mal est tellement repoussé que la partie ne finit jamais... et comme le jeu rend plus facile l'apprentissage au fur et à mesure du temps (les précédents étudiants donnent des boosts d'apprentissages aux suivants), tant que je continue d'accepter des étudiants (il n'y a pas de bouton "abandonner" ou similaire), le seigneur du mal n'a aucune chance. Tous mes manas débordent comme l'or, je suis à 350 de prestige et franchement je me suis lassé avant de monter davantage mais ça peut monter à l'infini. Bref, l'administrateur de l'école peut partir à la retraite, remplacé par un robot acceptant toutes les candidatures. Dans n'importe quelle classe. Car il y a tellement de bonus passif qu'ils peuvent même apprendre jusqu'à la maîtrise rien que dans la cafétariat, en interdisant l'apprentissage de toutes les magies ça doit encore tourner.

Quelque part c'est fun de pouvoir tester la partie en mode gros bill, quelque part c'est quand même un petit soucis d'équilibrage. L'avancée du seigneur du mal ne devrait-elle pas être exponentielle avec le temps, au lieu de linéaire? Ca résoudrait le problème...

**Comment by Neelan22** [comment]
Oui, on a déjà un peu affiné la limite mais pas assez. On va remodifier la formule pour la prochaine mise à jour. Techniquement, la partie ne peut pas ne jamais finir (formule en exponentielle), mais clairement il faut limiter plus que ça la barrière supérieure.
